ross/fluid_flow/tilting_pad.py

```diff
@@ -0,0 +1,1606 @@
+                            Mat_coef[k - 1, k - 1] = Ap
+                            Mat_coef[k - 1, k] = Ae
+                            Mat_coef[k - 1, k - 2] = Aw
+                            Mat_coef[k - 1, k + self.ntheta - 1] = As
+                            Mat_coef[k - 1, k - self.ntheta - 1] = An
+                            Mat_coef[k - 1, k + self.ntheta] = Ase
+                            Mat_coef[k - 1, k + self.ntheta - 2] = Asw
+                            Mat_coef[k - 1, k - self.ntheta] = Ane
+                            Mat_coef[k - 1, k - self.ntheta - 2] = Anw
+
+                        if kj == self.nX - 1 and ki > 0 and ki < self.nN - 1:
+                            Mat_coef[k - 1, k - 1] = Ap + Ae
+                            Mat_coef[k - 1, k - 2] = Aw
+                            Mat_coef[k - 1, k + self.ntheta - 1] = As + Ase
+                            Mat_coef[k - 1, k - self.ntheta - 1] = An + Ane
+                            Mat_coef[k - 1, k + self.ntheta - 2] = Asw
+                            Mat_coef[k - 1, k - self.ntheta - 2] = Anw
+
+                        if kj == 0 and ki == self.nN - 1:
+                            Mat_coef[k - 1, k - 1] = Ap + As - Aw
+                            Mat_coef[k - 1, k] = Ae + Ase
+                            Mat_coef[k - 1, k - self.ntheta - 1] = An - Anw
+                            Mat_coef[k - 1, k - self.ntheta] = Ane
+                            b[k - 1] = (
+                                b[k - 1]
+                                - 2 * Tmist[ki] * Aw
+                                - 2 * Tmist[ki - 1] * Anw
+                                - Tmist[ki + 1] * Asw
+                            )
+
+                        if ki == self.nN - 1 and kj > 0 and kj < self.nX - 1:
+                            Mat_coef[k - 1, k - 1] = Ap + As
+                            Mat_coef[k - 1, k] = Ae + Ase
+                            Mat_coef[k - 1, k - 2] = Aw + Asw
+                            Mat_coef[k - 1, k - self.ntheta - 1] = An
+                            Mat_coef[k - 1, k - self.ntheta] = Ane
+                            Mat_coef[k - 1, k - self.ntheta - 2] = Anw
+
+                        if ki == self.nN - 1 and kj == self.nX - 1:
+                            Mat_coef[k - 1, k - 1] = Ap + As + Ae + Ase
+                            Mat_coef[k - 1, k - 2] = Aw + Asw
+                            Mat_coef[k - 1, k - self.ntheta - 1] = An + Ane
+                            Mat_coef[k - 1, k - self.ntheta - 2] = Anw
+
+                        kj = kj + 1
+
+                    kj = 0
+                    ki = ki + 1
+
+                ki = 0
+                nn = 0
+
+                matcoef_test = np.loadtxt(
+                    "/home/verg1lio/Desktop/ross/ross/fluid_flow/matcoef.txt"
+                )
+                b_test = np.loadtxt("/home/verg1lio/Desktop/ross/ross/fluid_flow/b.txt")
```

Comment:
We do not need them, these were validation files, and did their job just fine. I'm keeping the testing code there commented out, just in case we need to do the same task once merged into the main ROSS code.